nemo_curator/utils/split_large_files.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Thanks for adding this, can we add some quick tests? 
If we you want to test out ray functionality and need a ray cluster you can use the fixture `shared_ray_client` (defined in conftest.py)

If I were you, I'd just test the python functions `(_split_table, _write_table_to_file  and split_parquet_file_by_size)` and assume thing work on ray..

Reply:
Just added some test that check file splitting is reasonable for parquet files with various row group sizes (both splitting row group and merging row group cases)